### PR TITLE
text.split implemented

### PR DESCRIPTION
**Pull request by EugeneDar**
https://github.com/objectionary/eo-strings/issues/22

### CODE REVIEW

**Comment by EugeneDar** [comment]
@Graur @yegor256 please have a look

File: src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOsplit.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+/**
+ * EO org.eolang.txt package.
+ *
+ * @checkstyle PackageNameCheck (4 lines)
+ * @since 0.0
```
Comment (Graur) [inline src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOsplit.java:29]: @EugeneDar I think we can change this version

**Review by Graur — CHANGES_REQUESTED** [review]

File: src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOsplit.java (hunk 1 above)
Comment (EugeneDar) [reply src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOsplit.java:29]: @Graur I will use 0.0.3

**Comment by EugeneDar** [comment]
@Graur please, have a look again

**Comment by EugeneDar** [comment]
@Graur please, have a look again

**Review by Graur — APPROVED** [review]
@EugeneDar Good one!
@yegor256 Please, take a look

**Comment by EugeneDar** [comment]
@yegor256 please have a look

**Comment by yegor256** [comment]
@EugeneDar there are merge conflicts. please, resolve them

**Comment by EugeneDar** [comment]
@yegor256 please, have a look again

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**